app/assets/stylesheets/forever_style_guide/modules/_nav-dropdowns-ambassador.scss

```diff
@@ -1,26 +1,18 @@
-$your-ambassador-intro-text-width: 110px;
+$ambassador-name-tab-max-space: 140px; //max width so that right nav won't overlap with logo
```

Comment:
Why not use width in this variable name?
ex. $ambassador-name-width might not be as explicit as name-tab, but I think it does the job just as well in a bit more concise manner.